onnxruntime/contrib_ops/cuda/bert/packed_multihead_attention.cc

```diff
@@ -41,7 +42,7 @@ PackedMultiHeadAttention<T>::PackedMultiHeadAttention(const OpKernelInfo& info)
 
   scale_ = info.GetAttrOrDefault<float>("scale", 0.0f);
 
-#if USE_FLASH_ATTENTION
+#if USE_MEMORY_EFFICIENT_ATTENTION
```

Comment:
Please also add a similar way to disable flash attention v2 using environment variable for testing purpose.